domain/postal_code/postal_code.py

```diff
@@ -0,0 +1,17 @@
+MAINLAND_DEPARTEMENT_CODE_LENGTH = 2
+OVERSEAS_DEPARTEMENT_CODE_LENGTH = 3
+OVERSEAS_DEPARTEMENT_IDENTIFIER = '97'
+
+
+class PostalCode:
+    postalCode: str
+
+    def __init__(self, postalCode: str):
+        self.postalCode = postalCode
+
+    def get_departement_code(self):
+        return self.postalCode[:OVERSEAS_DEPARTEMENT_CODE_LENGTH] if self._is_overseas_departement() \
+            else self.postalCode[:MAINLAND_DEPARTEMENT_CODE_LENGTH]
+
+    def _is_overseas_departement(self):
```

Comment:
Tu peux préciser le type de retour
```suggestion
    def _is_overseas_departement(self) -> bool:
```